client/js/helpers/parse.js

```diff
@@ -79,23 +80,20 @@ function parse(createElement, text, message = undefined, network = undefined) {
 	// arrays of objects containing start and end markers, as well as metadata
 	// depending on what was found (channel or link).
 	const channelPrefixes = network ? network.serverOptions.CHANTYPES : ["#", "&"];
-	const userModes = network?.serverOptions?.PREFIX.symbols || ["!", "@", "%", "+"];
```

Comment:
Is this change necessary? `network?.` should be same as doing `network ? network...`